src/main/resources/config/liquibase/changelog/20210104190000_changelog.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+    <changeSet author="stefanwaldhauser" id="20210104190000-1">
```

Comment:
can you please merge the two changelogs?

Reply:
Done in 4cee37886852187f7f4bf81550be2f73a41f3739.